django/core/management/base.py

```diff
@@ -95,7 +95,7 @@ class DjangoHelpFormatter(HelpFormatter):
     """
     show_last = {
         '--version', '--verbosity', '--traceback', '--settings', '--pythonpath',
-        '--no-color',
+        '--no-color', '--force_color',
     }
```

Comment:
Should this be `--force-color`?

Reply:
Thanks, done in a4d8e412e0295ac7d40bb87ee6e5f44649f97816.